サイトを日本語に翻訳する

martinheidegger opened the pull request:
今のところはNodeSchoolの東京のサイトが英語です。日本人もわかるように日本語へ翻訳したほうがいいでしょうか？

KunimitsuMasuda commented:
可能であれば日本語があった方が良いかと思います。英語が苦手な方もいるかと思うので。。

sotayamashita commented:
> 今のところはNodeSchoolの東京のサイトが英語です。日本人もわかるように日本語へ翻訳したほうがいいでしょうか？
> 
> 可能であれば日本語があった方が良いかと思います。英語が苦手な方もいるかと思うので。。

東京の[ページ](https://www.androidify.com/ja/)を翻訳しようと思います。ブランチは名は`feature/japanese`としてやっていこうと思います

https://github.com/sota0805/tokyo/tree/feature/japanese

sotayamashita committed:
Create editorconfig file

sotayamashita committed:
Create gitattributes file

sotayamashita committed:
Start to translate

martinheidegger commented:
PRに変更ました。

martinheidegger commented on index.html:
```diff
@@ -43,7 +43,7 @@
         <div class="shell">
           <div class="workshopper">
-            <code>» HELLO WORLD（こんにちは世界）</code>
+            <code>» HELLO WORLD（ハローワールド）</code>
```
https://github.com/maxogden/learnyounode-jp/blob/jp/exercises/menu.json#L2

sotayamashita committed:
Create README.ja.md

I created README.ja.md because Japanese don not like reading
English but I think we need original version for other nodeschool
contributers.

sotayamashita committed:
Add a each READEME link

Commit:
Fix a typo

Commit:
Add gitter badge

Commit:
Add gitter badge

sotayamashita committed:
Delete join chat section
